mkosi/installer/__init__.py

```diff
@@ -55,6 +55,7 @@ def mounts(cls, context: Context) -> list[Mount]:
         mounts = [
             *finalize_crypto_mounts(context.config),
             Mount(context.packages, "/work/packages"),
+            Mount(context.artifacts, "/work/artifacts"),
```

Comment:
This doesn't really fit here. I'd add it to each script function for now individually, or you can try and refactor the script functions a bit so they reuse more code, up to you.

Reply:
I was very torn about adding that there, because for the most part the content will not be packages, but I also don't want to say it shouldn't be packages. I'll add it to the script invocations for now.